tests/integration.rs

```diff
@@ -943,6 +943,62 @@ fn read_multiple_result_sets() {
     assert_eq!(2, second_vals.value(0));
 }
 
+#[test]
+fn read_multiple_result_sets_with_second_no_schema() {
```

Comment:
In order to test this it is enough to have only one SQL statement which yields an empty result. Why I appreciate the context of why somebody would execute an SQL statement which does not return anything IMHO the test intention becomes clearer if we do not muddy the water with `more_results`.

Reply:
Unfortunately due to a limitation of `odbc-api` expecting a result set with a non-empty schema for the first results set this fails. This is a workaround to have the cursor created to hold the handle to get the multiple results.


This test fails for this case:
```rust
#[test]
fn read_result_set_with_no_schema() {
    // Given a statement that is result-free
    let conn = ENV
        .connect_with_connection_string(MSSQL, Default::default())
        .unwrap();
    let cursor = conn
        .execute(
            "SELECT 1 AS A INTO #local_temp_table;",
            (),
        )
        .unwrap()
        .unwrap();

    // When
    // We have a cursor with no schema
    let reader = OdbcReaderBuilder::new()
        .with_max_num_rows_per_batch(1)
        .build(cursor)
        .unwrap();
    let schema = reader.schema();

    // Then
    // Check we have a schema with no fields
    let column_count= schema.fields().len();
    assert_eq!(0, column_count);
}
```

Reply:
oh yeah, I can see why you would need more results here.